src/client/components/hiluk-menu/hiluk-menu.tsx

```diff
@@ -91,7 +99,7 @@ export class HilukMenu extends React.Component<HilukMenuProps, HilukMenuState> {
       onClick={this.openRawDataModal.bind(this)}
     >{STRINGS.viewRawData}</li>);
 
-    if (getVisualizationDataset) {
+    if (getVisualizationDataset()) {
```

Comment:
if there's no visualization doesn't it make sense for the export option to not show up?